rita_client/src/exit_manager/mod.rs

```diff
@@ -255,46 +251,51 @@ fn decrypt_exit_state(
     Ok(decrypted_exit_state)
 }
 
-fn send_exit_setup_request(
+async fn send_exit_setup_request(
     exit_pubkey: WgKey,
     to: SocketAddr,
     ident: ExitClientIdentity,
-) -> impl Future<Item = ExitState, Error = RitaClientError> {
+) -> Result<ExitState, RitaClientError> {
     let endpoint = format!("http://[{}]:{}/secure_setup", to.ip(), to.port());
+    info!("Into this func 1");
     let ident = encrypt_exit_client_id(&exit_pubkey.into(), ident);
+    info!("Into this func 2");
```

Comment:
can we remove this log, or make it more verbose if it will be good to have in the future.

Reply:
Fixed, left these in by mistake